app/main/posts/posts-module.js

```diff
@@ -11,13 +11,15 @@ angular.module('ushahidi.posts', [])
 .directive('postAddForm', require('./detail/post-add-form.directive.js'))
 .directive('postValue', require('./detail/post-value.directive.js'))
 .directive('postMediaValue', require('./detail/post-media-value.directive.js'))
+.directive('postVideoValue', require('./detail/post-video-value.directive.js'))
 .service('PostActionsService', require('./common/post-actions.service.js'))
 
 // Create / Edit Screens
 .service('PostEntity', require('./modify/post-entity.service.js'))
 .service('PostEditService', require('./modify/post-edit.service.js'))
 .service('MediaEditService', require('./modify/media-edit.service.js'))
 .directive('postMedia', require('./modify/post-media.directive.js'))
+.directive('postVideo', require('./modify/post-video.directive.js'))
```

Comment:
I'd really like to improve the naming scheme on these. When we have PostVideo and PostVideoValue its not clear which is input or output. Could we have PostVideoInput/PostVideoEdit and PostVideoView?